demo/demo.js

```diff
@@ -143,7 +143,9 @@ map3.pm.addControls({
     drawMarker: true,
     drawPolygon: true,
     editPolygon: true,
-    drawPolyline: true,
+    drawPolyline: {
```

Comment:
the toolbar evaluation which buttons should be shown still works when there is an `Object` passed instead of `true` I assume?

Reply:
This is a cool way to pass options to the toolbar!
But I don't think they get applied right now - this would need work inside `Toolbar.js` to pass these options to the triggered functions (like Draw.enable).
Also would need documentation in the `README.MD`

Reply:
sorry, i left for vacations, but i'll look at these once I come back! thanks for the tips.

Reply:
ok I looked a bit into this and perhaps it would be better to somehow refactor this whole code. because from what i'm seeing is that you have some global settings that apply to all actions (e.g.  snapping, dragging). also for edit button I noticed this:
 ```
afterClick: () => {
                this.map.pm.toggleGlobalEditMode({
                    snappable: true,
                    draggable: true,
                });
            },
```
.. which I would understand that it activates snapping and dragging for editing always. now, if we provided some object setting for individual tool ,we could fine-tune whether a tool needs snapping, dragging etc. or not. 
but since this involved much work, I would correct errors for now and maybe create this stuff in a separate PR.